graspy/match/gmp.py

```diff
@@ -161,6 +164,14 @@ def fit(self, A, B, seeds_A=[], seeds_B=[]):
         Returns
         -------
         self : returns an instance of self
+
+        Attributes
+        -------
+        n_iter_ : Number of iterations run.
+
+        score_ : Objective function value.
+
+        perm_inds_ : Permutation Indicies
```

Comment:
No need to list the attributes here; you can delete this.

Reply:
Resolved!